subprojects/gradle/ci-logger/src/main/kotlin/com/avito/utils/logging/Gradle.kt

```diff
@@ -26,10 +28,35 @@ internal object CILoggerRegistry {
 
 private fun provideLogger(project: Project, loggerName: String): CILogger {
     return if (project.buildEnvironment is BuildEnvironment.CI && !project.buildEnvironment.inGradleTestKit) {
+        val endpoints: List<String>? = project.properties["avito.elastic.endpoints"]?.toString()?.split("|")
```

Comment:
I consider make a logger plugin that we will apply to the root project and configure our loggers there

Reply:
me too, i don't know how to approach such refactoring properly right now; let's separate this tasks